lib/Tool/Serialize.php

```diff
@@ -102,6 +102,7 @@ protected static function loopFilterCycles($element)
             $propCollection = get_object_vars($clone);
 
             foreach ($propCollection as $name => $propValue) {
+                $name = trim($name);
                 $clone->$name = self::loopFilterCycles($propValue);
```

Comment:
I took a second look and found out that the right approach is by using a `try` `catch` as found [here](https://github.com/pimcore/pimcore/issues/11801#issuecomment-1082784978)

```suggestion
                try{
                    $clone->$name = self::loopFilterCycles($propValue);
                } catch (\Throwable $e) {
                    continue;
                }
```

The `trim` works but is a false positive, it creates extra properties (5 instead of the expected 3)

![image](https://user-images.githubusercontent.com/6014195/210268337-ad7222f0-40b2-433f-9dc4-2a5bf106cf44.png)

Live example 
https://3v4l.org/U1u9p#v8.1.11  try catch
https://3v4l.org/52MFj#v8.1.11  trim

Reply:
This should work, too:

```suggestion
                if (!str_starts_with($name, "\0")) {
                    $clone->$name = self::loopFilterCycles($propValue);
                }
```

Reply:
Fine with me too :-)